dialog-el.html

```diff
@@ -6,11 +6,19 @@
     :host {
       position: absolute;
       display: inline-block;
+      transform: translate(0,0);
```

Comment:
With modals being broken due to the changes introduced in Norman-Amy@c3183fe I would go back through those changes and vet whether or not the actually need to exist to accomplish the feature.